goverge/main.py

```diff
@@ -214,7 +214,7 @@ def _parse_args(argv):
     p.add_argument(
         '--xml_dir',
         action='store',
-        default="xml_reports/",
+        default=os.getcwd() + "/xml_reports/",
```

Comment:
Probably slightly better to use `os.path.join` here though pretty minor unless you want this to be platform independent